frontend/src/components/pages/VerifyEmail.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext, useEffect } from "react";
+import { Flex, Image, VStack, Center, Text } from "@chakra-ui/react";
+import { useQuery } from "@apollo/client";
+import { Redirect } from "react-router-dom";
+import RHSLogo from "../../assets/RHSlogo.png";
+import AuthContext from "../../contexts/AuthContext";
+import { GET_USER_WITH_VERIFICATION_STATUS_BY_EMAIL } from "../../APIClients/queries/UserQueries";
+import { MANAGE_COURSES_PAGE } from "../../constants/Routes";
+import { AuthUser } from "../../types/AuthTypes";
+import authAPIClient from "../../APIClients/AuthAPIClient";
+
+const VerifyEmail = (): React.ReactElement => {
+  const { authUser, setAuthUser } = useContext(AuthContext);
+  const { refetch } = useQuery<
+    { userWithVerificationStatusByEmail: Omit<AuthUser, "accessToken"> },
+    { email: string }
+  >(GET_USER_WITH_VERIFICATION_STATUS_BY_EMAIL, { skip: true });
```

Comment:
Maybe add a comment about why we have this line for people who aren't familiar with what `skip: true` does.